pkg/storkctl/clusterpair.go

```diff
@@ -534,12 +589,16 @@ func newCreateClusterPairCommand(cmdFactory Factory, ioStreams genericclioptions
 	return createClusterPairCommand
 }
 
-func generateClusterPair(name, ns, ip, port, token, configFile, projectIDMappings string, reverse bool) (*storkv1.ClusterPair, error) {
+func generateClusterPair(name, ns, ip, port, token, configFile, projectIDMappings string, reverse bool, ignoreStorageOptions bool) (*storkv1.ClusterPair, error) {
 	opts := make(map[string]string)
-	opts["ip"] = ip
-	opts["port"] = port
-	// extract token from px-endpoint command
-	opts["token"] = token
+	if !ignoreStorageOptions {
+		opts["ip"] = ip
+		opts["port"] = port
+		// extract token from px-endpoint command
+		opts["token"] = token
+	} else {
+		opts["insert_storage_options_here"] = ""
```

Comment:
If the `ignoreStorageOptions` is going to be set only for Sync DR, then we can skip setting any opts fields, since they will not be used anyways

Reply:
done